lib/Echidna/Exec.hs

```diff
@@ -106,8 +106,8 @@ scoveragePoints = sum . fmap (S.size . S.map fst)
 -- | Capture the current PC and codehash. This should identify instructions uniquely (maybe? EVM is weird).
 pointCoverage :: (MonadState x m, Has VM x) => Lens' x CoverageMap -> m ()
 pointCoverage l = use hasLens >>= \v ->
-  l %= M.insertWith (const . S.insert $ (v ^. state . pc, Success)) (fromMaybe (W256 maxBound) $ h v) mempty where
-    h v = v ^? env . contracts . at (v ^. state . contract) . _Just . codehash
+  l %= M.insertWith (const . S.insert $ (v ^. state . pc, Success)) (fromMaybe BS.empty $ h v) mempty where
```

Comment:
I know it's been like that before, but I'm curious what is the scenario when `fromMaybe` defaults to `BS.empty`? Is this a valid state?

Reply:
I believe `fromMaybe` will never use it.

Reply:
It might be good to crash the program instead if that's the "impossible" case

Reply:
Done.